Externals/wxWidgets3/CMakeLists.txt

```diff
@@ -818,23 +818,20 @@ set(SRCS_UNIXGTK
 	"src/unix/uiactionx11.cpp"
 	"src/unix/utilsx11.cpp")
 
-include_directories(.)
-include_directories(include)
-
-set(SRCS
-	${SRCS_AUI}
-	${SRCS_COMMON}
-	${SRCS_GENERIC})
+add_library(wx STATIC ${SRCS_AUI} ${SRCS_COMMON} ${SRCS_GENERIC})
```

Comment:
I don't think we would want anything different for bundled libraries, it's better to be explicit.
Some of them require different build flags when used in shared mode.